domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -175,25 +169,48 @@ public String sqlFile(String tenantId, boolean tenantExists, TenantAttributes te
         });
   }
 
-  /**
-   * @param tenantAttributes parameters like module version that may influence generated SQL
-   * @return the SQL commands to create or upgrade the tenant's schema
-   * @throws IllegalArgumentException when templates/db_scripts/schema.json doesn't exist
-   * @throws TemplateException when processing templates/db_scripts/schema.json fails
-   */
-  public Future<String> sqlFile(Context context, String tenantId, TenantAttributes tenantAttributes) {
-    return tenantExists(context, tenantId)
-    .compose(tenantExists -> sqlFile(context, tenantId, tenantAttributes, tenantExists));
+  Future<Void> saveJob(TenantJob tenantJob, String tenantId, String jobId, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantId) + ".rmb_job";
+    String sql = "INSERT INTO " + table + " VALUES ('" + jobId + "', '" + Json.encode(tenantJob) + "'::JSONB)";
+    return postgresClient(context).execute(sql).mapEmpty();
+  }
+
+
+  Future<TenantJob> getJob(String tenantId, String jobId, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantId) + ".rmb_job";
+    String sql = "SELECT jsonb FROM " + table + " WHERE id = '" + jobId + "'";
+    return postgresClient(context).selectSingle(sql)
+        .compose(reply -> {
+          if (reply == null) {
+            return Future.failedFuture("Job not found " + jobId);
+          }
+          JsonObject o = reply.getJsonObject(0);
+          return Future.succeededFuture(o.mapTo(TenantJob.class));
+        });
+  }
+
+  Future<Void> updateJob(TenantJob tenantJob, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantJob.getTenant()) + ".rmb_job";
+    String sql = "UPDATE " + table
+        + " SET jsonb = '" + Json.encode(tenantJob) + "'::JSONB"
+        + " WHERE id = '" + tenantJob.getId() + "'";
+    return postgresClient(context).execute(sql).mapEmpty();
+  }
+
+  Future<Void> removeJob(String tenantId, String jobId, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantId) + ".rmb_job";
+    String sql = "DELETE FROM  " + table + " WHERE id = '" + jobId + "'";
+    return postgresClient(context).execute(sql).mapEmpty();
   }
 
   /**
-   * Installs or upgrades a module for a tenant.
-   *
-   * <p>The <code>handler</code> signals an error with a failing result and a {@link ResponseException}.
-   *
-   * @see <a href="https://github.com/folio-org/raml-module-builder#extending-the-tenant-init">Extending the Tenant Init</a>
-   *      for usage examples
-   */
+ * Installs or upgrades a module for a tenant.
+ *
+ * <p>The <code>handler</code> signals an error with a failing result and a {@link ResponseException}.
+ *
+ * @see <a href="https://github.com/folio-org/raml-module-builder#extending-the-tenant-init">Extending the Tenant Init</a>
+ *      for usage examples
+ */
```

Comment:
star indentation

Reply:
ok. old stuff.

Reply:
I mean the indentation of these 7 lines.

Reply:
Two spaces have been removed from the beginning of the seven lines. This should be reverted so that the stars at the beginning of the javadoc lines have the usual indentation.

Reply:
OK . Now as in master.